docs/guides/clustering-and-scaling/How-to-deploy-Ant-Media-Server-with-Helm.md

```diff
@@ -0,0 +1,96 @@
+- Ingress controller
+
+Add the AMS repository to Helm then install it as follows.
+
+```
+helm repo add antmedia https://ant-media.github.io/helm
+helm repo update
+helm install antmedia antmedia/antmedia --set origin={origin}.{example.com} --set edge={edge}.{example.com} --namespace antmedia --create-namespace
+```
+After the installation is finished, 1 MongoDB pod, 1 Ant Media Origin pod, 1 Ant Media Edge pod, and Nginx Ingress will be installed (Go to the bottom of the page for available parameters.) and the output of **kubectl get pods -n antmedia** will be as follows.
+
+```
+NAME READY STATUS RESTARTS AGE ant-media-server-edge-7d8fd58f94-dwqbs 1/1 Running 0 2m15s ant-media-server-origin-57d974f4f7-655rf 1/1 Running 0 2m15s antmedia-ingress-nginx-controller-6b49f64bfc-zbblx 1/1 Running 0 2m15s mongo-69888cbbb9-d2zrc 1/1 Running 0          2m15s
+```
+If the installation went as expected, run **kubectl get ingress -n antmedia** command to get your Ingress IP address and then update your DNS according to the ingress IP address and hostnames.
+
+`kubectl get ingress -n antmedia`
+
+**Example Output**
+
+```
+NAME                      CLASS    HOSTS                   ADDRESS        PORTS     AGE
+ant-media-server-origin   <none>   origin.antmedia.cloud   x.x.x.x        80, 443   9m45s
+ant-media-server-edge     <none>   edge.antmedia.cloud     x.x.x.x        80, 443   9m55s
+```
+You can do a DNS query as follows.
+```
+dig origin.antmedia.cloud +noall +answer
+dig edge.antmedia.cloud +noall +answer
+```
+Example output:
+```
+root@murat:~# dig edge.antmedia.cloud +noall +answer
+edge.antmedia.cloud.	300	IN	A	x.x.x.x
+```
+If the result of this output is your Ingress IP address, your DNS has been updated so you can access via HTTPS (self-signed) or HTTP.
+
+### Install SSL
+
+By default, a self-signed certificate comes in the Ant Media Server Kubernetes structure that you install with Helm. If you want, you can replace it with your own certificate as below or follow the steps below for Let's Encrypt.
+```
+kubectl create -n antmedia secret tls ${CERT_NAME} --key ${KEY_FILE} --cert ${CERT_FILE} 
+Use Let's Encrypt
+```
+If you want, you can do this with the script we have prepared below or a step-by-step installation.
+```
+wget https://raw.githubusercontent.com/ant-media/helm/add_helm_repo/ams-k8s-ssl.sh
+bash ams-k8s-ssl.sh
+```
+Then wait for the certificate to be created.
+
+If everything went well, the output of the **kubectl get -n antmedia** certificate command will show the value **True** as follows.
+```
+NAME                   READY   SECRET                 AGE
+antmedia-cert-origin   True    antmedia-cert-origin   21m
+antmedia-cert-edge     True    antmedia-cert-edge     24m
+```
+Then you can reach the Ant Media Edge/Origin instances over HTTPS.
+
+https://{origin}.{example}.{com}
+https://{edge}.{example}.{com}
+
+### Parameters
+
+You can customize the Ant Media Cluster installation using the following parameters.
```

Comment:
Is this still a work in progress @muratugureminoglu??

I see, sentence is saying the following parameters?